docs/API/plain_rules.md

```diff
@@ -396,23 +396,7 @@ the Perseo configuration). The `parameters` map includes the following fields:
     -   UPDATE: update attributes, asumming they exist (otherwise the update operation fails at CB)
     -   DELETE: delete attributes (or the entity itself if the attributes list is empty)
 -   trust: optional, trust for getting an access token from Auth Server which can be used to get to a Context Broker
-    behind a PEP. A trust is a way of Keystone to allow an user (trustor) delegates a role to another user (trustee) for
-    a subservice. Complete info could be found at:
-    -   [Trusts concept](https://docs.openstack.org/keystone/stein/user/trusts)
-    -   [Trusts API](https://docs.openstack.org/keystone/stein/api_curl_examples.html#post-v3-os-trust-trusts)
-    -   [Trust token flow example](./trust_token.md)
--   authentication: optional, authentication (host, port, user, password and service) configuration values that will be
-    used by updateAction rule (instead of default authentication defined by configuration) which will be used when a
-    trust token should be negotiated. i.e.:
-    ```json
-    "authentication": {
-        "host": "ext-keystone",
-        "port": 5001,
-        "user": "mycepuser",
-        "password": "myceppassword",
-        "service": "mycepuserservice"
-     }
-    ```
+    behind a PEP.
```

Comment:
This modification means that `authentication` field is no longer supported?

In addition, the usage of the new configTrust.js file should be documented (although not sure in which .md file...)

Reply:
Witch this changes authentication is used partially, just host and port.
configTrust could be documented in a newAproachTrust.md ?

Reply:
doc was updated by https://github.com/telefonicaid/perseo-fe/pull/745/commits/fd58e1862cb7486f83a4708debcacbc490c88f06